### PR TITLE
KDS-1760-migrate-table

### PR DESCRIPTION
**Pull request by gyfchong**
Stories are a bit of a mess, but arguably better than before with real code examples. Sending individual ones to chromatic cause creating a stickersheet that essentially duplicates the stories is a pain.

### CODE REVIEW

File: packages/components/src/Input/InputSearch/InputSearch.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React, { InputHTMLAttributes, useRef } from "react"
 import classnames from "classnames"
 import { LoadingSpinner } from "@kaizen/loading-spinner"
+import { ClearButton } from "~components/ClearButton"
 import { SearchIcon } from "~components/Icon/SearchIcon"
 import { OverrideClassName } from "~types/OverrideClassName"
-import { ClearButton } from "../../ClearButton"
 import styles from "./InputSearch.module.scss"
```
Comment (gyfchong) [inline packages/components/src/Input/InputSearch/InputSearch.tsx:7]: Not entirely sure what happened here

File: packages/components/src/Input/InputSearch/InputSearch.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React, { InputHTMLAttributes, useRef } from "react"
 import classnames from "classnames"
 import { LoadingSpinner } from "@kaizen/loading-spinner"
+import { ClearButton } from "~components/ClearButton"
 import { SearchIcon } from "~components/Icon/SearchIcon"
 import { OverrideClassName } from "~types/OverrideClassName"
-import { ClearButton } from "../../ClearButton"
```
Comment (gyfchong) [inline packages/components/src/Input/InputSearch/InputSearch.tsx:6]: Unrelated to the PR, but it happened so I'm just going to leaving it 😄

File: packages/components/src/Input/InputSearch/InputSearch.tsx (hunk 1 above)
Comment (gyfchong) [inline packages/components/src/Input/InputSearch/InputSearch.tsx:7]: Unrelated to the PR, but it happened so I'm just going to leaving it 😄

**Review by mcwinter07 — COMMENTED** [review]
Do we need a story in the MDX to showcase the `checkable` prop functionality?

**Review by mcwinter07 — APPROVED** [review]
Looks hunky dory :shipit:

**Pull request merged**